packages/grafana-data/src/types/panel.ts

```diff
@@ -27,6 +27,8 @@ export interface PanelPluginMeta extends PluginMeta {
   hideFromList?: boolean;
   /** Sort order */
   sort: number;
+  /** Indicate the preferred height of this panel. */
+  preferredHeight?: number;
```

Comment:
It's typed as a number so I'm assuming this is an absolute value (pixels). Isn't the "preferred height" dependent on the device that Explore is viewed on? I'm worried we end up here with a setting that the user should have some sort of control over.

Reply:
yes the height would be absolute value (pixels). I believe that the other panels on the explore page also use absolve heights.